src/lambda-edge/shared/logger.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+/* istanbul ignore file */
+
+export enum LogLevel {
+    'none' = 0,
+    'info' = 10,
```

Comment:
Switch info and error (was a bug I fixed on master).
Should be:
info: 30
error: 10

Reply:
Sure will update.